101-azurefirewall-sandbox/README.md

```diff
@@ -0,0 +1,16 @@
+# Create AzureFirewall sandbox setup
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-azurefirewall-sandbox%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-azurefirewall-sandbox%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates a virtual network with 3 subnets (server subnet, jumpbox subet and AzureFirewall subnet), a jumpbox VM with public IP and RDP access, A server VM with only a private IP, UDR route to point to AzureFirewall for the ServerSubnet and an azure firewall with 1 sample application rule and 1 sample network rule.
+Azure Firewall is a managed cloud-based network security service that protects your Azure Virtual Network resources.
+It is a fully stateful firewall as a service with built-in high availability and unrestricted cloud scalability.
+You can centrally create, enforce, and log application and network connectivity policies across subscriptions and virtual network.
+Azure Firewall uses a static public IP address for your virtual network resources allowing outside firewalls to identify traffic originating from your virtual network.
+The service is fully integrated with Azure Monitor for logging and analytics.
+Azure Firewall is currently a managed public preview that you need to explicitly enable using the Register-AzureRmProviderFeature PowerShell command. Learn more at https://docs.microsoft.com/en-us/azure/firewall.
```

Comment:
add some line breaks in here, this is one paragraph

Reply:
Done